enterprise-metrics/main.libsonnet

```diff
@@ -156,6 +156,17 @@ local removeNamespaceReferences(args) = std.map(function(arg) std.strReplace(arg
       type=d.T.string,
     ),
     licenseSecretName: 'gem-license',
+    '#adminTokenSecretName':: d.val(
+      default=self.adminTokenSecretName,
+      help=|||
+        When generating an admin token using the `tokengen` target, the result is written to the Kubernetes
+        Secret `adminTokenSecretName`.
+        To retrive the token from Kubernetes using `kubectl`:
+        $ kubectl get secret gem-admin-token -o jsonpath="{.data.grafana-token}" | base64 --decode ; echo
```

Comment:
IIRC, the grafana-token is actually double base64 encoded, once thanks it being a value in a Kubernetes Secret and another time because Grafana provisioning expects the token to be base64 encoded. 

With that in mind, I think we should clarify the docs here to use the `.data.token` field instead which will give users a usable token. Or perhaps we just need to include some variation of the previous paragraph as additional docs?

Reply:
Updated the help text a bit. Wdyt?